### ISSUE TITLE
Manual user creation

### ISSUE THREAD

**Issue by drraccoony** [minnesota-furs/mnfursvolunteers#5]
Currently its not possible for a user to create a new user.

**Comment by Snail51** [comment]
what is meant by this? do we want admins to be able to arbitrarily make new users? i can create new users with no issue?

**Comment by drraccoony** [comment]
Admins to create users, yes.

**Comment by Snail51** [comment]
how are we differentiating between users and admins? do you want me to create a special admin-only page for admins to be able to do so?

**Comment by drraccoony** [comment]
The user model should have a migration created to add a bool of `isAdmin` to the model, with a default of 0. Then there'd be a method made on the user model to check if the user is admin, which would look like this in blade files:
```blade
@if($user->isAdmin)
   <!-- Show admin actions -->
@endif
```

or if the user itself logged in is admin, you can use the `Auth` facade to check if the logged in user is an admin, like this:

```blade
@if(Auth::user()->isAdmin)
   <!-- If the user is an admin -->
@endif
```
https://laravel.com/docs/11.x/authorization#via-blade-templates

**Comment by Snail51** [comment]
Excellent, that's exactly what i was thinking. You are asking me to create the function to do this but not actually implement it into the website yet, correct?

**Comment by drraccoony** [comment]
If you feel thats easiest for you to accomplish, I think you could easily make a branch for this feature, and create a migration. Its up to you how confident you feel on this.

We can use "Web routes" middleware to stop certain actions/routes to users that aren't admins. https://laravel.com/docs/11.x/authorization#via-middleware

**Comment by drraccoony** [comment]
Resolved with MR #16
